ui/src/app/pages/workspace/workspace-about.spec.tsx

```diff
@@ -325,4 +327,75 @@ describe('WorkspaceAbout', () => {
       workspace.namespace
     );
   });
+
+  it('renders Community Workspace section when enabled', () => {
+    serverConfigStore.set({
+      config: { ...defaultServerConfig, enablePublishedWorkspacesViaDb: true },
+    });
+    component();
+    expect(screen.getByText('Community Workspace')).toBeInTheDocument();
+  });
+
+  it('does not render Community Workspace section when enablePublishedWorkspacesViaDb disabled', () => {
+    serverConfigStore.set({
+      config: { ...defaultServerConfig, enablePublishedWorkspacesViaDb: false },
+    });
+    component();
+    expect(screen.queryByText('Community Workspace')).not.toBeInTheDocument();
+  });
+
+  it('Publish button is enabled for workspace owner', () => {
```

Comment:
Do you mind changing the title of this test to mention that the workspace is not published.